test/e2e/config/vsphere-dev.yaml

```diff
@@ -0,0 +1,180 @@
+---
```

Comment:
Can we pull this file from CAPV repo? Otherwise we have to maintain all these versions

Reply:
for this file `vsphere-dev`, no. because it is e2e config specific to this project. for other provider manifests, yes we can use script to download them. e.g. `test/e2e/data/infrastructure-vsphere/v1*`

Reply:
Then better to call out which CAPV version or tag we are referring to